content/en/docs/concepts/services-networking/service.md

```diff
@@ -91,23 +110,29 @@ spec:
       targetPort: 9376
 ```
 
-This specification creates a new Service object named "my-service", which
+That manifest creates a new Service named "my-service", which
```

Comment:
do we want to clarify this is a `ClusterIP` type if Service, and that is why it gets an IP assigned?

Reply:
If this PR isn't introducing a regression, I recommend saving that for a follow-up. I don't want this PR to become “make Service and Ingress docs perfect”.